pom.xml

```diff
@@ -97,31 +97,31 @@
         <jetty.version>9.4.53.v20231009</jetty.version>
         <clojure.tools.logging.version>0.2.3</clojure.tools.logging.version>
         <carbonite.version>1.6.0</carbonite.version>
-        <snakeyaml.version>2.0</snakeyaml.version>
+        <snakeyaml.version>2.2</snakeyaml.version>
         <httpclient.version>4.5.14</httpclient.version>
         <httpclient.core.version>4.4.16</httpclient.core.version>
         <jctools.version>2.0.1</jctools.version>
         <jgrapht.version>0.9.0</jgrapht.version>
-        <guava.version>32.1.3-jre</guava.version>
+        <guava.version>33.0.0-jre</guava.version>
         <auto-service.version>1.0-rc4</auto-service.version>
-        <netty.version>4.1.100.Final</netty.version>
+        <netty.version>4.1.107.Final</netty.version>
         <sysout-over-slf4j.version>1.0.2</sysout-over-slf4j.version>
-        <log4j.version>2.21.1</log4j.version>
-        <slf4j.version>1.7.36</slf4j.version>
+        <log4j.version>2.23.0</log4j.version>
+        <slf4j.version>2.0.12</slf4j.version>
         <metrics.version>3.2.6</metrics.version>
         <mockito.version>4.11.0</mockito.version>
         <zookeeper.version>3.9.1</zookeeper.version>
         <snappy.version>1.1.10.4</snappy.version>
         <jline.version>0.9.94</jline.version>
         <hive.version>3.1.3</hive.version>
-        <hadoop.version>3.3.6</hadoop.version>
+        <hadoop.version>3.4.0</hadoop.version>
```

Comment:
Looks like `3.4.0` purged `hadoop-auth` ? Isn't listed in https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-auth anymore, so might be gone and needs migration?

Reply:
The changes for 3.4.0 are not documented on https://hadoop.apache.org/docs/stable/ 
Will revert for now, we can work out whether hadoop-auth is needed later

Reply:
looks like it is still at the RC level
https://github.com/apache/hadoop/tags